app/src/main/java/com/quran/labs/androidquran/QuranAdvancedPreferenceActivity.java

```diff
@@ -82,7 +83,7 @@ public void requestWriteExternalSdcardPermission(String newLocation) {
           REQUEST_WRITE_TO_SDCARD_PERMISSION);
     } else {
       // in the future, we should make this a direct link - perhaps using a Snackbar.
-      Toast.makeText(this, R.string.please_grant_permissions, Toast.LENGTH_SHORT).show();
+      ToastCompat.makeText(this, R.string.please_grant_permissions, Toast.LENGTH_SHORT).show();
```

Comment:
It's not possible for us to change the colors without getting a reference to the Toast itself because:

1) the attribute it uses for the background is private
2) it doesn't use attributes for the text color - it's hardcoded